index.html

```diff
@@ -4239,10 +4239,10 @@ <h3>Token Binding</h3>
 man-in-the-middle attack</a>. Any protocol using the Verifiable Credentials
 Data Model that requires protection against these kinds of attacks needs to
 perform some sort of token binding, such as
-<a href="https://tools.ietf.org/html/draft-ietf-tokbind-protocol-18">
-The Token Binding Protocol v1.0</a>, which ties the request for a
-<a>verifiable presentation</a> with the response. Any protocol that does not
-perform token binding is susceptible to man-in-the-middle attacks.
+<a href="https://tools.ietf.org/html/rfc8471"> The Token Binding Protocol
```

Comment:
We should probably use the informative reference syntax of ReSpec now that this is an RFC, please change to:

[[?RFC8471]]